agent/bench-scripts/pbench-fio.md

```diff
@@ -89,6 +89,22 @@ and then randomly read/write only an insignificant fraction of the entire set of
 yet still get representative throughput and latency numbers.  
 The same is true for using fio for random I/O on block devices.  
 
+## shared filesystems
+
+A slightly different set of parameters is needed for pbench-fio to work with shared filesystems (where same set of files
+is shared by multiple clients/mountpoints).  For this to work, you must do 2 things:
+
+* specify --target=my-directory 
+* specify --jobfile=/opt/pbench-agent/bench-scripts/templates/fio-shared-fs.job  
```

Comment:
@portante this needs to be --job-file to match the options processing

Reply:
Also, if --file-size or -s is not used, the fio job will not work [when using the fio-shared-fs.job], so we should probably add a third requirements here:

specify --file-size=size-in-MB

Reply:
Fixed by #1256.

Reply:
Should we elaborate on why this is?  Is the idea that the user must specify a file size that is <= to the available space divided by the number of clients?

Reply:
Well, now that I looked at how the fio job file is being generated, the `fs-shared-fs.job` does not contain a default `size` value.  I'll post a PR that we could consider.  We might not need to add this file size if we provide a default.

Reply:
Yes, adding a default size would be good, if we already do this for the regular job file

Reply:
@k-rister, let's review #1257 first, and then see what we should have for documentation.